antelope_contracts/contracts/deposit_proxy/src/deposit_proxy.cpp

```diff
@@ -0,0 +1,30 @@
+#include <eosio/symbol.hpp>
+#include <deposit_proxy.hpp>
```

Comment:
The `deposit_proxy.hpp` file should be located in `antelope_contracts/contracts/deposit_proxy/include/deposit_proxy/`. That way here we would say:

```cpp
#include <deposit_proxy/deposit_proxy.hpp>
```

And it would be very clear where the header file would be coming from.

That is the pattern we want to follow to not get too confused as we get many different modules that may depend on each other.